ShubhamPalriwala/cal.com
# feat: integrate formbricks in help feedback box

## What does this PR do?
The Help Feedback Box currently sends the data to our email service. This PR sends it to a pre-created Formbricks Survey for easier visibility and availability in the Formbricks App.

![image](https://github.com/ShubhamPalriwala/cal.com/assets/55556994/08ef123d-3840-4680-8166-e25e2857c88a)

We use the `@formbricks/api` package for minimal dependencies and create a response in the app.

## Requirement/Documentation
We introduce a new environment variable that needs to be set:
`NEXT_PUBLIC_FORMBRICKS` in the form of `[apiHost]+[environmentId]+[surveyId]`

For testing, use:
```
https://app.formbricks.com+cloob69in56synz0fg6l5vqmx+cloob6fn356u0nz0fl0mm1wa5
```

Before merging to main, 
- [ ] Create a Survey on Formbricks
- [ ] Create 2 questions (1 Open Text with Question ID as: `formbricks-share-comments-question` and 1 Rating question with Range of 4  and ID as `formbricks-rating-question` )

## Type of change


## How should this be tested?

- Set the env variable as suggested above
- Fill the Feedback Box inside the Help Button
- View in the Formbricks App and the response should be reflected

## Mandatory Tasks
